website/source/docs/providers/vsphere/r/virtual_machine.html.markdown

```diff
@@ -74,6 +74,8 @@ The following arguments are supported:
 * `dns_suffixes` - (Optional) List of name resolution suffixes for the virtual network adapter
 * `dns_servers` - (Optional) List of DNS servers for the virtual network adapter; defaults to 8.8.8.8, 8.8.4.4
 * `network_interface` - (Required) Configures virtual network interfaces; see [Network Interfaces](#network-interfaces) below for details.
+* `wait_for_network_interfaces` - (Optional) Specifies if we should wait for at least one network interface to appear and not only for the guest ip address
```

Comment:
I don't know of a use cases where we would not want the interfaces 😄  So I would default everything to true.  But the docs need to spell this out clearly.